app/src/main/java/com/beemdevelopment/aegis/ui/MainActivity.java

```diff
@@ -75,6 +75,7 @@
     private DatabaseManager _db;
     private boolean _loaded;
     private String _checkedGroup;
```

Comment:
Can you remove this unused field? <sup><sup>(please don't hate me)</sup></sup>

Reply:
I *just* noticed that too. Somehow my Android Studio's 'refactor' feature didn't actually rename the field itself. Will fix it.